cxx/assets/test_files/test_crosscat_bernoulli.schema

```diff
@@ -0,0 +1,4 @@
+col1 ~ bernoulli(id)
```

Comment:
Doesn't the relation name here have to match the second column of the .obs file?  Here it is "col1" but the .obs file uses "has_col1".

Reply:
Yeah this was a bad push. Fixed here and elsewhere.